packages/documentation-framework/components/example/example.js

```diff
@@ -109,7 +111,9 @@ export const Example = ({
     // These 2 are in the bundle anyways for the site since we dogfood
     ...reactCoreModule,
     ...reactTableModule,
-    ...(source === 'react-next' ? reactCoreNextModule : {})
+    ...(source === 'react-next' ? reactCoreNextModule : {}),
+    ...(source === 'react-deprecated' ? reactCoreDeprecatedModule : {}),
+    ...(source === 'react-deprecated' ? reactTableDeprecatedModule : {})
```

Comment:
I think we can use the condition `source === 'react-deprecated'` just once for the inclusion of both of these deprecated modules.